code/headers/joystick_value.hpp

```diff
@@ -1,6 +1,45 @@
 #pragma once
+#include <hwlib.hpp>
 
-struct joystick_value_s {
+class joystick_value_c {
```

Comment:
Just FYI, structs can overload operators so this does not have to be a class and since it does not even have a special constructor I suggest to change this back to a struct. This will probably come up if reviewed by a code reviewer when this goes to the master as is.

Reply:
Agree, changed